src/components/ui/hook-form/checkbox-group/index.tsx

```diff
@@ -0,0 +1,109 @@
+            }
+            .ant-checkbox-inner {
+                border-color: ${COLORS.blue};
+            }
+            &:not(.ant-checkbox-wrapper-disabled):hover
+                .ant-checkbox-checked:not(.ant-checkbox-disabled)
+                .ant-checkbox-inner {
+                background-color: ${COLORS.blue};
+            }
+
+            .ant-checkbox .ant-checkbox-inner {
+                & {
+                    width: 26px;
+                    height: 26px;
+                }
+                &:after {
+                    width: 8px;
+                    height: 13px;
+                }
+            }
+
+            .ant-checkbox-checked + span {
+                color: ${COLORS.blue};
+            }
+            .ant-checkbox + span {
+                font-size: 18px;
+                line-height: 1.3;
+            }
+        }
+    }
+`;
+
+type CheckboxGroupBaseProps = {
+    options: { key: string; label: string; value: string }[];
+    gutter?: Gutter | [Gutter, Gutter];
+    label?: string;
+} & UseControllerProps;
+
+export const CheckboxGroup = ({
+    name,
+    rules,
+    options,
+    label,
+    gutter = [24, 24],
+    ...props
+}: CheckboxGroupBaseProps) => {
+    const {
+        control,
+        formState: { errors },
+    } = useFormContext();
+
+    const error = errors[name] ? <>{errors[name]?.message}</> : '';
+
+    return (
+        <Flex vertical>
+            {label && <Label>{label}</Label>}
+            <Controller
+                name={name}
+                control={control}
+                rules={rules}
+                render={({ field: { value: arrayValue, onChange } }) => (
+                    <CheckboxAntd.Group
+                        {...props}
+                        value={arrayValue.map(
+                            (optionValue: { value: string }) =>
+                                optionValue.value,
+                        )}
+                        style={{ width: '100%' }}
+                        onChange={(checkedValues) => {
+                            const checked = options.filter((option) =>
+                                checkedValues.includes(option.value),
+                            );
+                            console.log(checked);
+                            onChange(checked);
+                        }}
+                    >
+                        <Row gutter={gutter}>
+                            {options.map((option) => (
+                                <Col key={option.key} span={12}>
+                                    <Checkbox value={option.value}>
+                                        {option.label}
+                                    </Checkbox>
+                                </Col>
+                            ))}
+                        </Row>
+                    </CheckboxAntd.Group>
+                )}
+            />
+            {error && <ErrorMessage motionId={name}>{error}</ErrorMessage>}
```

Comment:
Actually this is not required, because this is not required fields. But can let it be for consistancy and future where it might be needed. But we have two options:

1) remove errors where it is not needed right now
2) add AnimatePresence in each component. 

I think 1 will be better. Let's add it to a backlog. In general component looks good. I'll wait for your changes that we discussed at the call.